### PR TITLE
Add mkdocs

### PR DESCRIPTION
**Pull request by sidhulyalkar**

### CODE REVIEW

File: docs/src/citation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+If your work uses Element Session, please cite the following manuscript and Research
+Resource Identifier (RRID).
+
++ Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
+  Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
+  Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
+
++ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
+  Element Session (version `<Enter version number>`)
```
Comment (kabilar) [inline docs/src/citation.md:11]:
```suggestion
+ DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
  Element Event (version `<Enter version number>`)
```

**Review by kabilar — CHANGES_REQUESTED** [review]
Thanks @sidhulyalkar.  Please add content to the `index` page and update the `readme`.  Please mirror the latest updates to the `Concepts` page from [element-deeplabcut](https://datajoint.com/docs/elements/element-deeplabcut/0.2/concepts/).

File: docs/src/citation.md (hunk 1 above)
Comment (CBroz1) [reply docs/src/citation.md:11]: Please also add [automated versioning to the citation page](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2)

File: docs/src/citation.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 Resource Identifier (RRID).
 
 + Yatsenko D, Nguyen T, Shen S, Gunalan K, Turner CA, Guzman R, Sasaki M, Sitonic D,
   Reimer J, Walker EY, Tolias AS. DataJoint Elements: Data Workflows for
   Neurophysiology. bioRxiv. 2021 Jan 1. doi: https://doi.org/10.1101/2021.03.30.437358
 
 + DataJoint Elements ([RRID:SCR_021894](https://scicrunch.org/resolver/SCR_021894)) -
-  Element Session (version `<Enter version number>`)
\ No newline at end of file
+  Element Event (version `0.1.2`)
```
Comment (CBroz1) [inline docs/src/citation.md:11]: Please add automated versioning here. See [this commit](https://github.com/datajoint/element-deeplabcut/commit/352c44f98325cb79a249026d84310bff8417e7e2) for details

**Comment by CBroz1** [comment]
@sidhulyalkar The `nav.html` and `make_pages.py` files are empty

**Comment by CBroz1** [comment]
Outdated by #28

**Pull request closed without merging**